File: day 14/scope.cs
using System;
using System.Linq;

class Difference {
    private int[] elements;
    public int maximumDifference;

	// Add your code here
    public Difference(int[] v){
        elements = v;
    }

    public void computeDifference(){
        Array.Sort(elements);
        maximumDifference = Math.Abs(elements.Last() - elements[0]);
    }

} // End of Difference Class

class Solution {
    static void Main(string[] args) {
        Convert.ToInt32(Console.ReadLine());
        
        int[] a = Console.ReadLine().Split(' ').Select(x=>Convert.ToInt32(x)).ToArray();
        
        Difference d = new Difference(a);
        
        d.computeDifference();
        
        Console.Write(d.maximumDifference);
    }
}

File: day 14/scope.py
"""Get largest absolute difference between 2 numbers in a list.

Also exploring scope.
"""

class Difference: # pylint: disable=too-few-public-methods
    """Hold elements and max difference, and a function to compute difference."""

    def __init__(self, a):
        """Initialise the __elements list and maximumDifference."""
        self.__elements = a
        self.maximum_difference = 0

    def compute_difference(self):
        """Sort the list and taking the first and last values."""
        self.__elements.sort()
        self.maximum_difference = abs(self.__elements[-1] - self.__elements[0])

if __name__ == "__main__":
    _ = input()
    ARRAY = [int(ELEMENT) for ELEMENT in input().split(' ')]

    DIFFERENCE = Difference(ARRAY)
    DIFFERENCE.compute_difference()

    print(DIFFERENCE.maximum_difference)
